lib/context-menu.coffee

```diff
@@ -21,6 +21,14 @@ module.exports = ->
           'command': 'git-plus-context:diff'
         },
         {
+          label: 'Git diff branches',
```

Comment:
These commands need to be registered with atom just like the other commands in `git-plus.coffee` around Line 151.